javaScriptLoops-forLoop/practiceProblems/log24.js

```diff
@@ -0,0 +1,3 @@
+for(let i=2; i<=24; i+=2) {
```

Comment:
Nonstandard spacing. Please add a space after the `for`, and every comparison operator, addition/subtraction assignment and variable assignment:
```
for (let i = 2; i <= 24; i += 2)
```

Reply:
@nicoRDL, I saw the updates, but `i+=2` was left as is.

Reply:
> @nicoRDL, I saw the updates, but `i+=2` was left as is.

@nicoRDL, I see only one thing left to resolve ^

Reply:
Got left out on the last commit, my apologies. PR update with change requests

@lycoangelo 

https://github.com/nicoRDL/dotTheory-JS/pull/12/commits/782f4a8e486fbdf999564adb22d470d67f2e11df